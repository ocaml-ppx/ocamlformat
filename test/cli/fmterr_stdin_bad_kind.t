  $ echo 'module X : S' | ocamlformat --impl -
  ocamlformat: ignoring "<standard input>" (syntax error)
  File "<standard input>", line 2, characters 0-0:
  Error: Syntax error
  [1]
