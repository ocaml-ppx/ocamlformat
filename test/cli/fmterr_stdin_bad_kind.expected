ocamlformat: ignoring "<standard input>" (syntax error)
File "<standard input>", line 5, characters 0-0:
Error: Syntax error
